src/security/pci-guidance.md

```diff
@@ -0,0 +1,151 @@
+# PCI DSS Responsibility Guidance
+
+Platform.sh has many PCI certified customers using our services. Below we will attempt to clarify the PCI compliance responsibilities for Platform.sh and our customers.
+
+Simple audit questions can be handled via support tickets however in-depth support for your audit may required additional arrangements. Please contact your Platform.sh account manager to discuss your needs.
+
+
+## Overview
+
+Platform.sh provides a Platform as a Service (PaaS) solution which our customers may use to store, process, or transmit cardholder data.
+
+
+## Security & Compensating Controls
+
+ * For those customers needing to meet SAQ C or SAQ D, a Platform.sh Enterprise Plan may be needed in order to fully meet your needs. Please [contact us](https://platform.sh/contact) for more information.
```

Comment:
No leading space on the bullets.  I... don't now if you want them to be separate UL blocks or not.